x-pack/plugins/security/server/audit/audit_events.ts

```diff
@@ -202,6 +203,7 @@ const savedObjectAuditVerbs: Record<SavedObjectAction, VerbsTuple> = {
   saved_object_find: ['access', 'accessing', 'accessed'],
   saved_object_add_to_spaces: ['update', 'updating', 'updated'],
   saved_object_delete_from_spaces: ['update', 'updating', 'updated'],
+  saved_object_open_point_in_time: ['open', 'opening', 'opened'],
```

Comment:
NIT: I would go with

```
open pit
opening pit
opened pit
```

to be more explicit, as we did for `saved_object_remove_references` on the next line